### ISSUE TITLE
dialogs (e.g. settings in reader view): escape key does not trigger "close" action, enter/return should "accept" / okay button

### ISSUE THREAD

**Issue by danielweck** [readium/readium-desktop#62]

**Comment by clebeaupin** [comment]
Does not work on lcp passphrase dialog

**Comment by danielweck** [comment]
I am not sure why this issue was closed, the problem is 100% reproducible with the LCP info dialog, and even though other modal dialogs correctly implement the escape key closing, they do not focus the keyboard on a suitable default action button (so that the return / enter / space key can activate the button immediately, without having to keyboard-tab).
I suggest we re-open this issue.

**Comment by danielweck** [comment]
Note that keyboard-tabbing into the default dialog button is currently broken, because of:
https://github.com/readium/readium-desktop/issues/61
